backend/handlers/like_handler.py

```diff
@@ -48,6 +49,8 @@ def post(self, user, post_key, comment_id=None, reply_id=None):
         """Handle POST Requests."""
         """This method is only meant to give like in post, comment or reply and send notification."""
         post = ndb.Key(urlsafe=post_key).get()
+        Utils._assert(post.state == 'deleted',
```

Comment:
make tests for it, thus  you guarantee that any changes in this assert will be seen.

Reply:
Done.